package.json

```diff
@@ -15,7 +15,7 @@
     "framework/**/*"
   ],
   "dependencies": {
-    "@dvsa/mes-test-schema": "3.12.0",
+    "@dvsa/mes-test-schema": "3.21.0",
```

Comment:
Could we look into why this is a prod dependency? I.e. is it being used for anything other than type checking? If it isn't then it can be moved to devDependencies by running `npm i @dvsa/mes-test-schema@3.2.1 --save-exact --save-dev`

Reply:
Checked with tom doesn't seem like it is used for anything else so ill move it :)

Reply:
Checked with tom doesn't seem like it is used for anything else so ill move it :)